src/components/combobox-cities.tsx

```diff
@@ -138,8 +145,14 @@ export function ComboboxCities({ value, onChangeId, onCityChange }: ComboboxCiti
             <DrawerTrigger asChild>
                 <ComboboxButton />
             </DrawerTrigger>
-            <DrawerContent className='max-h-3/5 h-3/5 bg-white'>
-                <div className='mt-4 h-full border-t'>
+            <DrawerContent
+                className='bg-white'
+                style={{
```

Comment:
prefer tailwindcss rules over style definitions, you can use the cn() utility to combine logic with classnames and achieve same result

Reply:
the reason is that in the styles compilation step, that can cause race conditions and the results maybe not what expected and have different styles in development and in production, as style compiles different than tailwindcss styles